docs/hugo/content/introduction/multitenant-deployment.md

```diff
@@ -3,16 +3,43 @@ title: Deploying Azure Service Operator v2 in multi-tenant mode
 linktitle: Multitenancy
 ---
 
-The current release version of the Azure Service Operator (ASO) deployment YAML installs the operator in _single-tenant_ mode:
-deployed in the `azureserviceoperator-system` namespace with _a single set of Azure credentials_,
-and manages resources in _any namespace_ in the cluster.
-That single operator deployment handles webhooks fired when Azure resources are changed.
+Currently, we support two types of multitenancy with Azure Service Operator (ASO): 
+
+### Single operator multitenancy
+Single operator deployed in the `azureserviceoperator-system` namespace with _a single set of global Azure credentials_,
```

Comment:
minor: I think that you can remove the emphasis on "a single set of global Azure credentials" because now we actually support other creds.

Maybe say: 
```
Single operator deployed in the `azureserviceoperator-system` namespace.
This operator can be configured to manage resources with multiple different identities:
* Single global credential `aso-controller-settings` deployed as part of operator deployment.
* Per-namespace credential `aso-credential`. When this credential is created in a namespace, all resources in that namespace automatically use it instead of the global credential. This credential can be in a different tenant or subscription than the global credential.
* Per-resource credential, indicated by the `serviceoperator.azure.com/credential-from` annotation. This credential must be in the same namespace as the resource but can be in a different tenant or subscription than both the global credential and per-namespace credential.

When presented with multiple credential choices, the operator chooses the most specific one: per-resource takes precedence over per-namespace which takes precedence over global.
```

Reply:
Possibly chop a bit from the above since you do cover some of it down below in the deployment section.

I do think it's at least clarifying the 3 flavors up here though (global/namespace/per-resource) as that doesn't really have that much to do with deployment per-se.

Thoughts?

Reply:
Nice! Yeah, makes sense to explain up here.